src/features/agent-dashboard/components/AgentDetailForm.tsx

```diff
@@ -59,28 +69,16 @@ export const AgentDetailForm: FC<Props> = ({
     register,
     formState: { errors, isValid, isDirty, isSubmitting, isSubmitted },
     handleSubmit,
-    trigger,
     control,
-    watch,
     setError,
   } = useForm<FormData>({
     resolver: yupResolver(schema),
     mode: 'all',
     defaultValues: {
       ...defaultValues,
-      // Make state dropdown default to empty instead of first option + Ensure that the address2 confirms to the string type requirement
-      address: {
-        ...defaultValues.address,
-        state: defaultValues?.address?.state ?? '',
```

Comment:
At the moment, the state selection experience is inconsistent with the staging server when a new agent is being created. We should set the state's default value to an empty string. If we don't , then the associated Form.Select control will use the first value in the statelist, i.e. Alabama.

Reply:
Are you running the frontend with this branch on the backend `https://github.com/Shift3/dj_starter_demo/tree/chore/address-to-agent`?